packages/devtools_app/lib/src/ui/service_extension_elements.dart

```diff
@@ -181,8 +181,8 @@ class RegisteredServiceExtensionButton {
   }
 
   final RegisteredServiceDescription serviceDescription;
-  final VoidAsyncFunction action;
-  final VoidFunctionWithArg errorAction;
```

Comment:
this second one is a good one to replace.

Reply:
I was behind on my fork. This one was already dropped. I fixed the function type here though, no need to give an argument a meaningless name in the type definition...